.env

```diff
@@ -13,6 +13,8 @@ APPLICATION_THEME=orange
 
 #environment can be changed if it is toggled
 CREDENTIAL_REGISTRY_EDIT=true
+#databack can enable if it is toggled
```

Comment:
typo error
`data backup can be enabled if it is toggled`

Reply:
done